releases/stellar/2025-01-ITS-v1.0.0.md

```diff
@@ -0,0 +1,180 @@
+# STELLAR ITS v1.0.0
+
+|                | **Owner**                            |
+| -------------- | ------------------------------------ |
+| **Created By** | @ahramy <ahram@interoplabs.io>       |
+| **Deployment** | @RiceAndMeet <steven@interoplabs.io> |
+
+| **Network**          | **Deployment Status** | **Date** |
+| -------------------- | --------------------- | -------- |
+| **Devnet Amplifier** | -                     | TBD      |
+| **Stagenet**         | -                     | TBD      |
+| **Testnet**          | -                     | TBD      |
+| **Mainnet**          | -                     | TBD      |
+
+- [Crates](https://crates.io/users/interoplabs-ci)
+- [Releases](https://github.com/axelarnetwork/axelar-cgp-stellar/releases)
+
+## Background
+
+Changes in the release:
+
+1. This is the v1.0.0 initial ITS release.
+
+## Deployment
+
+Create an `.env` config.
+
+```yaml
+PRIVATE_KEY=xyz
+ENV=xyz
+CHAIN=stellar-2024-q4
+```
+
+```bash
+# Deploy Interchain Token
+node stellar/deploy-contract.js deploy interchain_token --version v1.0.0
+
+# Deploy Token Manager
+node stellar/deploy-contract.js deploy token_manager --version v1.0.0
+
+# Deploy Interchain Token Service
+node stellar/deploy-contract.js deploy interchain_token_service --version v1.0.0
```

Comment:
See the Sui ITS release PR, Stellar ITS needs to be registered on ITS Hub afterwards